CONTRIBUTING.md

```diff
@@ -52,3 +52,14 @@ guard
 ## Daru internals
 
 To get an overview of certain internals of daru and their implementation, go over [this blog post](http://v0dro.github.io/blog/2015/08/16/elaboration-on-certain-internals-of-daru/).
+
+
+## Troubleshooting
+
+Daru uses Travis CI for running and deploying the build. Since Travis is configured only for Ruby versions before Ruby-2.5.0, the test suite returns the following error when `bundle exec rspec` is run
```

Comment:
Actually you are misunderstanding the Travis CI. We are configuring the travis CI for automatic testing for Ruby versions 2.2, 2.3, 2.4 . Please read more about Travis CI/CD and github hooks, to understand it better.